kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java

```diff
@@ -486,10 +486,6 @@ public void testWorkItemOperations() throws Exception {
 
             processClient.abortWorkItem(CONTAINER_ID, processInstanceId, workItemInstance.getId());
 
-            workItems = processClient.getWorkItemByProcessInstance(CONTAINER_ID, processInstanceId);
```

Comment:
I think it would be better to keep it here. This call checks that there isn't any other work item left in process instance. Can be put in try/catch block.